### PR TITLE
Update release-1.16 branch config.toml for release v1.19

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Updated config.toml for release v1.19

/hold

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -95,40 +95,40 @@ js = [
...
+url = "https://kubernetes.io"
+
+[[params.versions]]
+fullversion = "v1.18.8"
 version = "v1.18"
-githubbranch = "v1.18.0"
+githubbranch = "v1.18.8"
 docsbranch = "release-1.18"
-url = "https://kubernetes.io"
+url = "https://1-18.docs.kubernetes.io"
```
Comment (sftim) [inline config.toml:109]:
```suggestion
url = "https://v1-18.docs.kubernetes.io"
```

**Review by sftim — COMMENTED** [review]
Hi. I spotted one typo. Everything else looks fine @savitharaghunathan

File: config.toml (hunk 1 above)
Comment (savitharaghunathan) [reply config.toml]: Thanks for the catch @sftim :)

**Comment by kbhawkey** [comment]
You may want to create different PR titles for the related config.toml pull requests, otherwise, it appears as if
the changes are identical. Something like: "Update release-1.16 config.toml for release 1.19"?

**Review by sftim — COMMENTED** [review]
/lgtm

**Comment by jimangel** [comment]
/lgtm
/approve
Per discussion [here](https://kubernetes.slack.com/archives/C1J0BPD2M/p1598466973088100?thread_ts=1598447979.070600&cid=C1J0BPD2M), I am manually merging this PR as part of the 1.19 release. 🎉

**Pull request merged**